core/src/main/groovy/noe/server/ServerAbstract.groovy

```diff
@@ -604,9 +617,21 @@ abstract class ServerAbstract implements IApp {
 
   /**
    * Check log files for ERRORS and WARNINGS
-   * TODO add offsets for logs (or delete logs before each test - replace symlinks)
    */
   List<String> verifyLogs() {
+    final List<String> defaultFilteredLines = platform.isFips() ? Arrays.asList(
```

Comment:
Are these lines only applicable for JWS? If they are, it IMHO doesn't make sense to have them in `ServerAbstract`.

Reply:
Re: the extraction of the pid:

> Thanks @PaulLodge . Exactly as you say. EAP uses set of scripts in $EAP_HOME/bin to start. Specifically standalone.sh and domain.sh.

See my answer to Paul [here](https://github.com/web-servers/noe-core/pull/126#discussion_r793117901). The script from jws is catalina.sh too, but what matters here is what "ps -ef" result contains. The .sh has disappeared after java is executed, which means that this whole code segment is harmful. I tested empirically. Could you do the same? i.e. do `ps -ef | grep 1471948789`  and `ps -ef | grep 1471948789 | grep .sh` while your servers are running to see if the result disappears, as is the case for jws.

If in your case grepping for .sh is useful (in our case destroys the result) then further changes in the code will be needed, as in our case the code is not finding any pid this way.